tests/test_misc.py

```diff
@@ -1,3 +1,8 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""pytests for :mod:`mutmut.cache`"""
```

Comment:
I'd rather rename the file to `test_cache` and remove this comment. Again shebang, encodings.